core/federated/RTI/rti_remote.c

```diff
@@ -1030,9 +1009,7 @@ void send_reject(int* socket_id, unsigned char error_code) {
     lf_print_warning("RTI failed to write MSG_TYPE_REJECT message on the socket.");
   }
   // Close the socket.
```

Comment:
Minor: how about adding more information for this call? For example, "Close the socket without reading until EOF."

Reply:
Thank you, I fixed it as suggested.